osu.Game/Screens/Play/Player.cs

```diff
@@ -33,14 +32,16 @@ public class Player : OsuScreen
 
         internal override bool ShowOverlays => false;
 
-        internal override bool HasLocalCursorDisplayed => !hasReplayLoaded && !IsPaused;
+        internal override bool HasLocalCursorDisplayed => !hasReplayLoaded && !IsPaused && !IsFailed;
 
         private bool hasReplayLoaded => hitRenderer.InputManager.ReplayInputHandler != null;
 
         public BeatmapInfo BeatmapInfo;
 
         public bool IsPaused { get; private set; }
 
+        public bool IsFailed { get; private set; }

```

Comment:
I don't know if this is a kind of code convention but shouldn't it be `HasFailed`